src/main/java/org/apache/commons/collections4/bloomfilter/package-info.java

```diff
@@ -56,20 +56,21 @@
  *
  *     <li>Extractor - The Extractors are {@code FunctionalInterfaces} that are conceptually iterators on a {@code BitMap}, an {@code Index}, or a
  *     collection of {@code Cell}s, with an early termination switch.  Extractors have
- *     names like {@code BitMapExtractor} or {@code IndexExtractor} and  have a {@code processXs} methods that take a
+ *     names like {@code BitMapExtractor} or {@code IndexExtractor} and have a {@code processXs} methods that take a
  *     {@code Predicate<X>} argument (e.g. {@code processBitMaps(LongPredicate)} or {@code processIndicies(IntPredicate)}).
  *     That predicate is expected to process each of the Xs in turn and return {@code true} if the processing should continue
  *     or {@code false} to stop it. </li>
  * </ul>
  *
- * <p>There is an obvious association between the BitMap and the Index in that if bit 5 is enabled in the BitMap than the index must contain the index 5.</p>
+ * <p>There is an obvious association between the BitMap and the Index in that if bit 5 is enabled in the BitMap than the indices must contain the
```

Comment:
This still does not address what you are referring to with the noun `Index`. We do not have an `Index` object and do not refer to this elsewhere in the code. Can we replace `Index` with `indices`?